src/main/java/net/minecraftforge/common/BiomeDictionary.java

```diff
@@ -328,6 +329,12 @@ private static void registerVanillaBiomes()
         addTypes(Biomes.SAVANNA_PLATEAU, HOT, SAVANNA, PLAINS, SPARSE, RARE, OVERWORLD, PLATEAU);
         addTypes(Biomes.BADLANDS, MESA, SANDY, DRY, OVERWORLD);
         addTypes(Biomes.WOODED_BADLANDS, MESA, SANDY, DRY, SPARSE, OVERWORLD, PLATEAU);
+        addTypes(Biomes.MEADOW, PLAINS, PLATEAU, OVERWORLD);
```

Comment:
Isn't this part of the mountain category?

Reply:
Yes, but it´s very low. Just goes up a little bit and the flattens out. Like a plateau.

Reply:
Could it perhaps makes sense to have both Plateau and Mountain? As it is indeed part of the vanilla mountain category, but it´s "shaped" like a plateau.